### DISCUSSION TITLE
poke_interval not working for EmrJobFlowSensor reschedule mode for Airflow version 5.1.0

### DISCUSSION THREAD

**Discussion by sonalprsd** [apache/airflow#32479] [Q&A]
I have provided poke_interval as 300 for EmrJobFlowSensor running in rescheduled mode. I see that the worker is able to run the task in rescheduled mode but the cluster is being poked every 3 s to check if it reached the target states WAITING or COMPLETED. Every 3 s, the task is being put to up_for_reschedule state . As a result, the airflow sensor task fails since retries are exhausted and EMR cluster is still getting bootstrapped. I observe the same behavior even on tweaking the poke_interval to 3000 or any other number.
I don't think this expected behavior for Airflow version 5.1.0. Can someone point out what am i missing?

Adding the code snippet below to provide an idea of all parameters being passed.

```
@task(task_id="job_flow_sensor", default_args=DEFAULT_ARGS)
def create_job_flow_sensor(common_params):
        context = get_current_context()
        cluster_id = context['ti'].xcom_pull(key='emr_cluster')['job_flow_id']
        EmrJobFlowSensor(
            task_id=f"check_cluster_{cluster_id}",
            job_flow_id=cluster_id,
            aws_conn_id=aws_connection,
            target_states=["COMPLETED", "WAITING"],
            failed_states=["TERMINATING", "TERMINATED", "TERMINATED_WITH_ERRORS"],
            mode='reschedule',
            poke_interval=300
        ).execute(context)
        return cluster_params
```

**Comment by eladkal** [comment]
Your code is operator inside operator.
Its a bad practice and does not do any sensor job.
Remove the task decorator and just use sensor directly.

See https://stackoverflow.com/a/65168908/14624409 for more information
